src/Xamarin.Android.Build.Tasks/Xamarin.Android.CSharp.targets

```diff
@@ -35,8 +35,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
              Disable generation to avoid "bizarre" build errors. -->
         <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
         <_AndroidResourceDesigner>Resource.designer.cs</_AndroidResourceDesigner>
-        <IsXBuild Condition="'$(MSBuildRuntimeVersion)' == ''">true</IsXBuild>
```

Comment:
xbuild?!? Looking at this file, it's really old and only imported by classic Xamarin.Android. Maybe we don't need to touch this file here, and we can delete all references to `Xamarin.Android.CSharp.targets` in a different PR?